src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheckTest.java

```diff
@@ -45,6 +45,7 @@ protected String getPackageLocation() {
     @Test
     public void testBadOverrideFromObject() throws Exception {
         final DefaultConfiguration checkConfig = createModuleConfig(MissingOverrideCheck.class);
+        checkConfig.addAttribute("javaFiveCompatibility", "false");
```

Comment:
we don't need to add default values explicitly

Reply:
I did think of this but https://github.com/checkstyle/checkstyle/pull/10106#discussion_r654936865

Reply:
done